src/Form/FormMapper.php

```diff
@@ -106,7 +111,7 @@ public function add($name, $type = null, array $options = [], array $fieldDescri
         );
 
         // Note that the builder var is actually the formContractor:
-        $this->builder->fixFieldDescription($this->admin, $fieldDescription, $fieldDescriptionOptions);
```

Comment:
don't think so, because in `Sonata` every implementation uses only 2 params (that's `*Builder` classes).

If some one breaks the rules, and passes 3 params to `Sonata\AdminBundle\Builder\BuilderInterface::fixFieldDescription` that their fault. 
`Sonata\AdminBundle\Builder\BuilderInterface::fixFieldDescription` strictly declares only 2 params

Reply:
Ok, I agree with you